src/uv-config.json

```diff
@@ -3,12 +3,40 @@
         "rightPanelEnabled": false
     },
     "modules": {
-        "footerPanel": {
-            "options": {
-                "downloadEnabled": false,
-                "fullscreenEnabled": true,
-                "shareEnabled": false
-            }
+      "headerPanel": {
+        "options": {
+          "settingsButtonEnabled": true
         }
+      },
+      "footerPanel": {
+          "options": 
+          {
+            "downloadEnabled": true,
+            "fullscreenEnabled": true,
+            "moreInfoEnabled": false,
+            "printEnabled": true,
+            "shareEnabled": false
+          },
+          "content": {
+            "bookmark": "Add to bookmarks",
+            "download": "Download ",
+            "embed": "Embed",
+            "feedback": "Feedback",
+            "exitFullScreen": "",
+            "fullScreen": "",
```

Comment:
Why are we removing the tooltip labels for 'fullScreen' and 'exitFullScreen'? And for that matter, do we need to include any of these labels? I'd think the UV defaults would be fine.